.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## [Jira ticket link](link here)
```

Comment:
This is an OS project, there's no need to have a ticket link. Most likely we only need a description for now.